### PR TITLE
Update to Stardog 6.2.1

### PR DESCRIPTION
**Pull request by joephayes**
Part of preparation for deploying Cross System Search

* Updates Stardog dependency from 6.0.1 to 6.2.1 (we are upgrading stardog instances on NASA VMs to 6.2.1)
* Updates Clojure to 1.9
* Misc. dependency updates
* Copyright updates

Ran tests locally and all pass.

### CODE REVIEW

File: project.clj [hunk 1]
```diff
@@ -13,15 +13,18 @@
 ;; WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 ;; See the License for the specific language governing permissions and
 ;; limitations under the License.
-(defproject stardog-clj "6.0.2"
+(require 'cemerick.pomegranate.aether)
```
Comment (joephayes) [inline project.clj]: I had to add these lines to get `lein` to work with the Stardog Maven Repo. Is it OK to leave them?

File: project.clj [hunk 2]
```diff
@@ -13,15 +13,18 @@
...
+(cemerick.pomegranate.aether/register-wagon-factory!
+ "http" #(org.apache.maven.wagon.providers.http.HttpWagon.))
+(defproject stardog-clj "6.2.1"
   :description "Stardog-clj: Clojure bindings for Stardog"
   :url "http://stardog.com"
   :license {:name "Apache License"
             :url "http://www.apache.org/licenses/LICENSE-2.0"}
-  :dependencies [[org.clojure/clojure "1.8.0"]
-                 [com.complexible.stardog/client-http "6.0.1" :extension "pom"]]
+  :dependencies [[org.clojure/clojure "1.9.0"]
```
Comment (joephayes) [inline project.clj]: Current version of Clojure is 1.10. I figured it was good to be no more than one minor version from the current version.

**Comment by joephayes** [comment]
They pass on my machine

```
13:50 $ lein midje
Warning: implicit middleware found: cider-nrepl.plugin/middleware
Please declare all middleware in :middleware as implicit loading is deprecated.
Warning: implicit middleware found: refactor-nrepl.plugin/middleware
Please declare all middleware in :middleware as implicit loading is deprecated.
SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
ERROR StatusLogger No log4j2 configuration file found. Using default configuration: logging only errors to the console. Set system property 'org.apache.logging.log4j.simplelog.StatusLogger.level' to TRACE to show Log4j2 internal initialization logging.
nil
All checks (33) succeeded.
✔ ~/dev/stardog-clj [jph-update-stardog-version|✔]
13:54 $

```

Do you have the testdb created? (https://github.com/stardog-union/stardog-clj#building-the-library)

**Comment by brianru** [comment]
Weird, I just got a very different less helpful error message here.

**Comment by joephayes** [comment]
@AlBaker - any special release steps I need to perform?

**Comment by AlBaker** [comment]
> @AlBaker - any special release steps I need to perform?

it is published on clojars, I can see about adding you so you can do releases

**Pull request merged**